service/history/workflowcache/cache.go

```diff
@@ -54,7 +54,7 @@ type wfCache struct {
 	domainCache            cache.DomainCache
 	metricsClient          metrics.Client
 	logger                 log.Logger
-	getCacheItemFn         func(domainID string, workflowID string) (*cacheValue, error)
+	getCacheItemFn         func(domainName string, domainID string, workflowID string) (*cacheValue, error)
```

Comment:
Why do we need both domainName and domainID?

Reply:
Good point, we can just use the name in the cache key, should be simpler, fixed